airflow/hooks/base.py

```diff
@@ -41,8 +41,17 @@ class BaseHook(LoggingMixin):
     object that can handle the connection and interaction to specific
     instances of these systems, and expose consistent methods to interact
     with them.
+
+    :param logger_name: Name of the logger used by the Hook to emit logs.
+        If set to `None` (default), the logger name will fall back to `{class.__module__}.{class.__name__}`
+        (e.g. DbApiHook will have *airflow.providers.common.sql.hooks.sql.DbApiHook* as logger).
```

Comment:
One NIT -> this docstring is now not entirely accurate (lack of "airflow.task.hooks" prefix.